### PR TITLE
BUG: fix some minor issues in the overview

### PR DESCRIPTION
**Pull request by ikerexxe**
- Change is for in
- Add missing apostrophe
- Change comma for period
- Rewrite sentence
- Change bracket for comma
- Rewrite whole paragraph to make it more understandable

Signed-off-by: Iker Pedrosa ipedrosa@redhat.com

### CODE REVIEW

File: src/selinux_overview.md [hunk 1]
```diff
@@ -68,17 +68,17 @@ and limited to the mimimum privileges required to do their job, so in a
     sandboxing services.
 9.  SELinux will not stop memory leaks or buffer over-runs (because its
     not designed to do this), however it may contain the damage that may
-    be done.
-10. SELinux will not stop all viruses/malware getting into the system
-    (as there are many ways they could be introduced (including by
+    be caused by this flaws.
```
Comment (pcmoore) [inline src/selinux_overview.md]: Should this be "... caused by these flaws."?

**Comment by pcmoore** [comment]
Hi @ikerexxe, thanks for the edits!  I had one small question (see above), let me know what you think.

File: src/selinux_overview.md (hunk 1 above)
Comment (ikerexxe) [reply src/selinux_overview.md]: Yes, it should be in the plural.

**Comment by ikerexxe** [comment]
Updated taking into account your comment.

**Comment by pcmoore** [comment]
Thanks for the quick update, I've gone and merged this via 38824442ff3d778ac39acef94ac9aa719e132b5a.

**Pull request closed without merging**